tests/e2e/test_rosacli_node_pool.go

```diff
@@ -859,4 +858,134 @@ var _ = Describe("Edit nodepool",
 				Expect(err).To(HaveOccurred())
 				Expect(rosaClient.Parser.TextData.Input(output).Parse().Tip()).To(ContainSubstring("Failed to get autoscaling or replicas: 'Autoscaling enabled on machine pool '%s'. can't set replicas'", machinepoolName))
 			})
+
+		It("create/describe machinepool with user tags for HCP - [id:73492]",
+			labels.High, labels.Day2, labels.NonClassicCluster,
+			func() {
+				By("Get the Organization Id")
+				rosaClient.Runner.JsonFormat()
+				whoamiOutput, err := rosaClient.OCMResource.Whoami()
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetFormat()
+				whoamiData := rosaClient.OCMResource.ReflectAccountsInfo(whoamiOutput)
+				organizationID := whoamiData.OCMOrganizationID
```

Comment:
In the future, I'd like there to be a common function to envelope `whoami` to get the user information. e.g. `rosaClient.OCMResource.UserInfo()`